pillar.example

```diff
@@ -8,14 +8,26 @@ hana:
   # you can also use this to a single node if need to differ. see hana2
   # Warning: only a unique solution can exist into a node.
   saptune_solution: 'HANA'
+  
+  # Specify the path to the sapcar executable & HANA database server installation sar archive
+  # The sar archive will be extracted to path specified at hdbserver_extract_dir (optional, by default /sapmedia/HANA)
+  # Make sure to use the latest/compatible version of sapcar executable, otherwise file may be extracted incorrectly
+  sapcar_exe_file: '/sapmedia/sapcar.exe'
+  hdbserver_sar_file: '/sapmedia/IMDB_SERVER_LINUX.SAR'
+  hdbserver_extract_dir: '/sapmedia/HANA'
+  # Or specify the path to already extracted HANA platform installation media
```

Comment:
I would put the `software_path` at the top of installation variables, and comment that this variable has prevalence over `hdbserver` variables